arch/configure.defaults

```diff
@@ -2066,6 +2066,59 @@ RLFLAGS         =
 CC_TOOLS        =      $(SCC)
 NETCDFPAR_BUILD	=      CONFIGURE_NETCDFPAR_BUILD
 
+###########################################################
+#ARCH    AMD Linux x86_64 #serial smpar dmpar dm+sm
```

Comment:
Usually this line includes some information about the compilers that are used in the configuration. Should something like `LLVM` or `Flang/Clang` or something similar be included?

Reply:
Hi @mgduda ,

Thanks for your comments. We have addressed your comments in the [d39e2ca](https://github.com/wrf-model/WRF/pull/1684/commits/d39e2cadaa7618a0ee8f7f03ce9dcf9a62776fce) .

Request you to do the needful.

Thank you

Reply:
Hi @mgduda and @weiwangncar,

Gentle Reminder.

It has been a long time since we have not received any update. Could you please look at [d39e2ca](https://github.com/wrf-model/WRF/pull/1684/commits/d39e2cadaa7618a0ee8f7f03ce9dcf9a62776fce) and confirm?

Thanks in advance.

Reply:
@amd-toolchain-support I just left a comment in the other thread on this PR regarding whitespace on line 45.